### ISSUE TITLE
Edit tips

### ISSUE THREAD

**Issue by Nattusia** [Nattusia/custom_tree#1]
1. show the edit tips for content types (what to write to each field)
2. and if it is possible to make them optional (I mean that when creating the content type, how could it be selected if the edit tips are shown or not)

**Comment by Nattusia** [comment]
Originally we have this tips. By anywhere reason they does not displayed in the Bootstrap theme which we use.
The only question is if should this tips be hidden and pop up when user over the "question sign", or they should be visible by default?

**Comment by Nattusia** [comment]
We do already. Drupal depends on jQuery at all.
Why it's not good? As I know jQuery UI does not supported anymore, but jQuery still on. (jQuery and jQuery UI are a little different libraries).

**Comment by mmmikko** [comment]
Yes, I meant JQuery UI. I got confused reading https://www.drupal.org/project/jquery_ui_tooltip , so https://www.drupal.org/project/beautytips could actually be good option. But I haven't yet seen it in action, so can't say really.

**Comment by Nattusia** [comment]
This module works wrong in D8. It has been ported wrong as I can see.
So I written some script for tooltips by myself.
If form element has a description you'll see the "info" icon opposite its name. http://task.sitiq.com/_screen/2020-06-18_224252.png
Clicking on it makes the tip visible.

**Comment by mmmikko** [comment]
Ok, let's use this for the time being.
